keith-vscode/src/constants.ts

```diff
@@ -13,6 +13,10 @@ export const views = {
         id: 'kieler-simulation-tree',
         name: 'KIELER Simulation Tree',
     },
+    checker: {
+        id: 'kieler-model-checker',
+        name: 'KIELER Model Checker',
+    }
```

Comment:
not table view specific

Reply:
i guess this is already part of the implementation for the model checker table view that was not excluded fully from this PR. Does not hurt anyone to already include it here.